tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -10,7 +10,7 @@
 
   <mksrf_flai>
     <entry>
-      <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.25x0.25.LUH2.histsimyr1850-2015.c20230226/mksrf_pftlaihgt_ctsm52_histLUH2_2005.c20230226.nc</data_filename>
+      <data_filename>/glade/campaign/cesm/development/lmwg/landuse_source_data/CTSM53RawData/globalctsm53histMKSRFDeg025_240709/mksrf_landuse_ctsm53_pftlai_CLIM.c240709.nc</data_filename>
```

Comment:
This and other new files need to be moved into inpudata data under lnd/clm2/rawdata and rimported. Since we are now confident in them we might as well do that.

Reply:
DONE `cp -r /glade/campaign/cesm/development/lmwg/landuse_source_data/CTSM53RawData /glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/rawdata/.`

TODO ./rimport -list

Reply:
Awesome.